src/components/movieGrid/grid.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './grid.css';
+import MovieItem from '../movieItem/item';
+import movieList from '../../movies';
+
+function MovieGrid() {
+  // Simulate Error for ErrorBoundary component
+  // throw new Error('Hello from error');
+
+  const [movies, setMovies]   = React.useState();
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const timeout = setTimeout(() => {
+      setMovies(movieList);
+      setLoading(false);
+    }, 2000);
+
+    return () => clearTimeout(timeout);
+  });
```

Comment:
Please, add a second argument to your `useEffect` hook. If you want it run just once (like simulating componentDidMount) just set it to an empty array.